hydrophone.go

```diff
@@ -141,8 +143,12 @@ func main() {
 		/*
 		 * hydrophone setup
 		 */
-	store := sc.NewMongoStoreClient(&config.Mongo)
-
+	storage, err := sc.NewStore(&config.Mongo, logger)
+	if err != nil {
```

Comment:
I would add a comment there to clearly say that we are only checking that databases configuration is not missing or malformed
 ```suggestion
       /* Check that database configuration is valid. It does not check database availability */
	if err != nil {
```